package.json

```diff
@@ -29,5 +35,8 @@
   ],
   "publishConfig": {
     "access": "public"
+  },
+  "dependencies": {
+    "jest": "^29.3.1"
```

Comment:
Since we use this for more than just Jest, this should probably be a `devDependency`?